addons/project/models/project_update.py

```diff
@@ -0,0 +1,199 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from datetime import timedelta
+from dateutil.relativedelta import relativedelta
+from werkzeug.urls import url_encode
+
+from odoo import api, fields, models
+
+STATUS_COLOR = {
+    'on_track': 10,  # green
+    'at_risk': 2,  # orange
+    'off_track': 1,  # red
+    'on_hold': 4,  # light blue
+}
```

Comment:
@tivisse Would you advice an other way to handle this dict ? (used to map the status into an integer color)